inject-generator/src/main/java9/module-info.java

```diff
@@ -0,0 +1,8 @@
+module io.avaje.inject.generator {
+
+  requires java.compiler;
+  requires java.annotation;
+  requires io.avaje.inject;
```

Comment:
Should there be a `requires javax.inject`?

Reply:
I'm going to say **_Yes_** (and add it).  It has been working for me without a `requires javax.inject` but that doesn't seem right - perhaps related to having an unnamed module (java.annotation) in there.  Hmmm.

Adding javax.inject as that seems more correct.  In theory javax.inject comes in with io.avaje.inject now but I've only just made that change and it was working before so confused as to why it was working.

Reply:
I think if you decide to declare javax.inject as transitive in io.avaje.inject, then it wouldn't be necessary to then require it here (since it would then be picked up from io.avaje.inject).

Reply:
Yes.  I added it thinking it wouldn't harm being explicit but now thinking I should really leave it up to io.avaje.inject to bring it in (and remove it again).